scipy/cluster/tests/test_hierarchy.py

```diff
@@ -948,7 +948,11 @@ def test_labels_as_array_or_list(self, xp):
         Z = linkage(xp.asarray(hierarchy_test_data.ytdist), 'single')
         labels = xp.asarray([1, 3, 2, 6, 4, 5])
         result1 = dendrogram(Z, labels=labels, no_plot=True)
-        result2 = dendrogram(Z, labels=list(labels), no_plot=True)
+        # imitating `tolist` in a standard-compliant way
+        labels_list = []
+        for i in range(labels.shape[0]):
+            labels_list += [int(labels[i])]
+        result2 = dendrogram(Z, labels=labels_list, no_plot=True)
```

Comment:
How about this @lucascolley?
```suggestion
        result2 = dendrogram(Z, labels=labels, no_plot=True)
```

Reply:
yep, much nicer